channels/drdynvc/client/drdynvc_main.c

```diff
@@ -1507,9 +1506,10 @@ static UINT drdynvc_virtual_channel_event_connected(drdynvcPlugin* drdynvc, LPVO
 
 	settings = (rdpSettings*)drdynvc->channelEntryPoints.pExtendedData;
 
-	for (index = 0; index < settings->DynamicChannelCount; index++)
+	for (index = 0; index < freerdp_settings_get_uint32(settings, FreeRDP_DynamicChannelCount);
+	     index++)
```

Comment:
`freerdp_settings_get_uint32(settings, FreeRDP_DynamicChannelCount);` should be part of another PR